### DISCUSSION TITLE
How to monitor used space on partitions?

### DISCUSSION THREAD

**Discussion by cjacobm** [fluent/fluent-bit#6954] [Q&A] [answered]
Hi, 

I My fluent-bit.conf I have:

[INPUT]
    Name disk
    tag  disk.local
    Interval_Sec 5

But, my output is this:
disk.local: [1678020620.794262218, {"read_size"=>0, "write_size"=>1146880}]

How to monitor used space on partitions? The disk´s input not support this feature?

Thank You!

**Comment by leonardo-albertovich** [comment, marked as answer]
You can get that from `node_exporter_metrics` with this configuration : 

```
[SERVICE]
    flush 1
    grace 1
    log_level info

[INPUT]
    name node_exporter_metrics
    # If you are running 2.0 you cannot filter which metrics are produced
    # but if you are running master you can uncomment the following line in 
    # order to get just the filesystem stats
    #metrics filesystem

[OUTPUT]
    name stdout
    match *
```

Just keep in mind that those are metrics, not logs.

**Reply by cjacobm** [reply]
It worked! Thank you very much!

**Comment by aspantel** [comment]
For some reason, node_exporter_metrics does not produce any metrics for me on Ubuntu.
How can I debug this? Thank you

**Reply by patrick-stephens** [reply]
Turn on debug `log_level`, check for warnings: https://chronosphere.io/learn/fluent-bit-tips-tricks/#Warnings%20are%20errors

Check permissions for `/proc`.
How are you running it - container/package? What's the config?

**Comment by aspantel** [comment]
No container is involved here, fluentbit was installed using apt on Ubuntu and is run using sysctl
All my inputs work fine (i.e. exec below) except the node_exporter_metrics. I have this config on 7,8 virtual servers and they all have this problem.

[INPUT]
    name node_exporter_metrics
    tag node_metrics
    metrics "meminfo,filesystem"
    
    [INPUT]
    Name          exec
    Tag           disk_space
    Command       df --output=avail,pcent,target --type=ext4
    Interval_Sec  60
    Interval_NSec 0
    
    [OUTPUT]
    Name            http
    Match           *
    Host            api.axiom.co
    Port            443
...



/proc permissions dr-xr-xr-x 192 root             root

**Comment by aspantel** [comment]
[ info] [input:node_exporter_metrics:node_exporter_metrics.3] thread instance initialized
[router] match rule node_exporter_metrics.3:http.0

**Reply by aspantel** [reply]
I see this but I don't know if it's related to node_exporter_metrics

[debug] [task] created task=0x7f6925c37860 id=1 without routes, dropping.
